packages/augur-ui/src/modules/contracts/actions/contractCalls.ts

```diff
@@ -27,7 +27,7 @@ import { NewMarket } from 'modules/types';
 import { TestNetReputationToken } from '@augurproject/core/build/libraries/GenericContractInterfaces';
 
 export function clearUserTx(): void {
-  const Augur = augurSdk.get();
+  // const Augur = augurSdk.get();
```

Comment:
this was causing issues when trying to switch accounts, augurSdk wasn't initialized.